.github/workflows/ci.yml

```diff
@@ -0,0 +1,131 @@
+name: CI
+
+on:
+  pull_request:
+    branches:
+      - '**'
+
+jobs:
+  build-docker:
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - container: wpilib/roborio-cross-ubuntu:2020-18.04
+            artifact-name: Athena
+            build-options: "-Ponlylinuxathena"
+          - container: wpilib/raspbian-cross-ubuntu:10-18.04
+            artifact-name: Raspbian
+            build-options: "-Ponlylinuxraspbian"
+          - container: wpilib/aarch64-cross-ubuntu:bionic-18.04
+            artifact-name: Aarch64
+            build-options: "-Ponlylinuxaarch64bionic"
+          - container: wpilib/ubuntu-base:18.04
+            artifact-name: Linux
+            build-options: ""
+    name: "Build - ${{ matrix.artifact-name }}"
+    runs-on: ubuntu-latest
+    container:
+      image: ${{ matrix.container }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build with Gradle
+        run: ./gradlew build -PbuildServer ${{ matrix.build-options }}
+      - uses: actions/upload-artifact@v1
+        with:
+          name: ${{ matrix.artifact-name }}
+          path: build/allOutputs
+
+  build-windows:
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - artifact-name: Win64
+            architecture: x64
+          - artifact-name: Win32
+            architecture: x32
+    name: "Build - ${{ matrix.artifact-name }}"
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v1
+        with:
+          java-version: 11
+          architecture: ${{ matrix.architecture }}
+      - name: Build with Gradle
+        run: ./gradlew build -PbuildServer
+      - uses: actions/upload-artifact@v1
+        with:
+          name: ${{ matrix.artifact-name }}
+          path: build/allOutputs
+
+  build-mac:
+    name: "Build - macOS"
+    runs-on: macos-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v1
+        with:
+          java-version: 11
+      - name: Build with Gradle
+        run: ./gradlew build -PbuildServer
+      - uses: actions/upload-artifact@v1
+        with:
+          name: macOS
+          path: build/allOutputs
+
+  build-cmake:
+    name: "Build - CMake"
+    runs-on: ubuntu-latest
+    container:
+      image: wpilib/roborio-cross-ubuntu:2020-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: configure
+        run: mkdir build && cd build && cmake -DWITHOUT_ALLWPILIB=OFF ..
+      - name: build
+        run: cd build && make -j3
+
+  wpiformat:
+    name: "wpiformat"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Fetch all history and metadata
+        run: |
+          git fetch --prune --unshallow
+          git branch master origin/master
```

Comment:
Turns out this breaks when you run CI on commits merged into master, but not on any other CI run.